CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
-v2.1.0 - 27 October 2020
+vX.Y.Z - TBD
```

Comment:
this seems not great

Reply:
This is because we don't know for sure what the next release is going to be or when.

Will we issue a quick bugfix release as 2.0.2 today? Will we hold onto this until 2.1.0 at some later date?

This is something that shouldn't be updated until the actual release is being rolled.